src/Dynamo.All.Net6.sln

```diff
@@ -330,6 +331,24 @@ Global
 		{F382C3F8-C55C-4350-800A-3D13A94F8E13}.Debug|NET60_Windows.ActiveCfg = Debug|NET60_Windows
 		{F382C3F8-C55C-4350-800A-3D13A94F8E13}.Debug|NET60_Windows.Build.0 = Debug|NET60_Windows
 		{F382C3F8-C55C-4350-800A-3D13A94F8E13}.Release|NET60_Windows.ActiveCfg = Release|NET60_Windows
+		{51BB6014-43F7-4F31-B8D3-E3C37EBEDAF4}.Release|x64.ActiveCfg = Release|x64
+		{51BB6014-43F7-4F31-B8D3-E3C37EBEDAF4}.Release|x64.Build.0 = Release|x64
+		{4CAB9005-0001-4B03-BBA0-57070ED105F1}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Please remove these other platforms from this sln, we only need `NET60_Windows` and `NET60_Linux`.
I think if you base your work on master first they will already be gone. @pinzart90 recently removed them I think.

Reply:
Only windows for all.net6
We do not support Linux for our UI

Reply:
yes, thanks for the correcttion @pinzart90 - though I think before @GuillermoRCADSK does more work here, lets decide if we can just delete this old tool.

Reply:
Done